test/test_get_ops.js

```diff
@@ -0,0 +1,132 @@
+var expect = require('chai').expect;
+var ShareDbMongo = require('..');
+var getQuery = require('sharedb-mingo-memory/get-query');
+var sinon = require('sinon');
+
+var mongoUrl = process.env.TEST_MONGO_URL || 'mongodb://localhost:27017/test';
+
+function create(callback) {
+  var db = new ShareDbMongo(mongoUrl, {
+    mongoOptions: {},
+    getOpsWithoutStrictLinking: true
+  });
+  db.getDbs(function(err, mongo) {
+    if (err) return callback(err);
+    mongo.dropDatabase(function(err) {
+      if (err) return callback(err);
+      callback(null, db, mongo);
+    });
+  });
+};
+
+require('sharedb/test/db')({create: create, getQuery: getQuery});
```

Comment:
I'm unsure if this is what's causing the CI test failure, but I did notice the number of test cases shot up a lot more than expected. I believe this is the cause, since it's re-including test cases that are already included by test_mongo.js.

Try removing this and see if it helps?

Reply:
took tests down to 702 cases (from 947)

Reply:
Push it up and let's see how the tests are on the CI runner, then!